interfaces/dotnet/Cantera/Cantera.csproj

```diff
@@ -41,11 +41,10 @@
 
   <Target Name="GenerateInterop" BeforeTargets="CoreCompile">
     <PropertyGroup>
-      <Script>$([MSBuild]::NormalizePath(../../../interfaces/sourcegen/run.py))</Script>
       <GeneratedPath>$([MSBuild]::NormalizePath($(IntermediateOutputPath)/sourcegen/))</GeneratedPath>
     </PropertyGroup>
 
-    <Exec Command="python3 $(Script) --api=csharp --output=$(GeneratedPath)"
+    <Exec Command="sourcegen --api=csharp --output=$(GeneratedPath) --root=../../.."
```

Comment:
I think this is probably a case where we want to use `python -m sourcegen`, for the same reason it's preferred to use `python -m pip` and make sure it's pulled from the right place, especially on Windows.